### ISSUE TITLE
[CDN] Multiple standalone files seem to be stuck in version 2.0.0

### ISSUE THREAD

**Issue by aguyinmontreal** [Semantic-Org/Semantic-UI#6532]
I'm currently using version 2.2.12 and the standalone CSS video component in the video.css seem to be different than the one in the the full semantic.css file.   

I didn't make a full diff check, however, for example, I can see that the `.ui.embed` block is missing, and the version seem to be stuck to 2.0.0.

It looks like this issue is affecting **multiple files** and for **multiple versions** from 2.0.0 to 2.3.3: colorize.js, colorize.min.js, video.css, video.min.css, video.js, video.min.js, visit.js and visit.min.js.

example: https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.12/components/video.css

**Comment by Banandrew** [comment]
Apparently, we still have `video` component in the `dist` of `master`, and I think that’s why cdnjs grabs it, not sure why `colorize`/`visit` are available for `2.3.3`, because both [don’t exist in cdnjs](https://github.com/cdnjs/cdnjs/tree/master/ajax/libs/semantic-ui/2.3.3/components) for `2.3.3`.

**Comment by aguyinmontreal** [comment]
My big question is why the video component **different** in the standalone and the semantic.css.

I was scratching my head a few days wondering why in hell my production and development css looked different when I found this.

I think the video/colorize/visit files should be removed/replaced for all versions since 2.0.0, not only for 2.3.3.

**Comment by ColinFrick** [comment]
@aguyinmontreal There is no video component in SUI2, there is the embed module for videos: https://semantic-ui.com/modules/embed.html

**Comment by aguyinmontreal** [comment]
@Banandrew @ColinFrick 

I understand you both say that there should not be a video component in SUI2. However, whether you like it or not, there **is** a video component in all full semantic.css and semantic.min.css files from version 2.0.0 to version 2.3.3.

Here are some examples:
- [For version 2.3.3](https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.3/semantic.css), it starts at line 33418 (starts with a `.ui.embed` block)
- [For version 2.2.12](https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.12/semantic.css), it starts at line 30822 (starts with a `.ui.embed` block)
-[For version 2.2.1](https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.1/semantic.css), it starts at line 30515 (starts with a `.ui.embed` block)

**What is more strange is that these video components are different than the ones found in the standalone video.css and video.min.css files.**

Here are some examples:
- [For version 2.3.3](https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.3/components/video.css), it starts with a `.ui.video` block
- [For version 2.2.12](https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.12/components/video.css), it starts with a `.ui.video` block
- [For version 2.2.1](https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.1/components/video.css), it starts with a `.ui.video` block

**And this is only for css, I didn't check js files.**

**Comment by ColinFrick** [comment]
Yes seems like the header is wrong, that is the embed module: https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.3/components/embed.css

**Comment by aguyinmontreal** [comment]
@Banandrew 
colorize/visit standalone seem no longer distributed in 2.3.3, however it was from 2.0.0 to 2.3.0

**Comment by aguyinmontreal** [comment]
@ColinFrick ok, thanks

**Comment by Banandrew** [comment]
@aguyinmontreal My bad, I was sure I was looking at `2.3.3` when I was typing the message—it doesn’t matter, though, we can’t remove the `video` component from `dist` without the maintainer, so we’ll just have to wait. Thanks again for bringing it up!

**Comment by Nazariy** [comment]
Is it possible to update CDN with most recent version, seems like it's still on version 2.3.3?

**Comment by ColinFrick** [comment]
@Nazariy there is an open issue for this: https://github.com/cdnjs/cdnjs/issues/12953
